setup.py

```diff
@@ -4,6 +4,8 @@
 
 from setuptools import setup, find_packages
 
+# pylint: disable=inconsistent-return-statements
```

Comment:
try to disable in the method `get_api_key` as:
`def get_api_key(client, username, secret):  # pylint: disable=R1710`